kotlin/hexagon/build.gradle

```diff
@@ -0,0 +1,22 @@
+
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.6.10"
+}
+
+ext {
+    gradleScripts = "https://raw.githubusercontent.com/hexagonkt/hexagon/2.0.0-B3/gradle"
```

Comment:
what are the rationals behind this ?

Reply:
Along with the toolkit, some Gradle scripts are also released to ease the projects configuration. I.e: https://raw.githubusercontent.com/hexagonkt/hexagon/2.0.0-B3/gradle/application.gradle would add utilities to create a single JAR with all the dependencies.

This variable is provided to be able to switch from one release to another one changing only the tag in the URL.

Reply:
I understand the idea (even I'm far away from `gradle` philosophy). will it not be clean (e.g -> more trust for developer) to use a `gradle` _plugin_ to define that ?